src/css/custom.css

```
@@ -13,103 +54,189 @@
   --ifm-color-primary-light: #4e88fc;
   --ifm-color-primary-lighter: #5c92fd;
   --ifm-color-primary-lightest: #89b0fd;
+  --ifm-font-family-base: 'Gilroy';
+  --ifm-font-family-monospace: 'Gilroy';
+  --ifm-heading-font-family: 'Gilroy-Medium';
+  --ifm-breadcrumb-item-background-active: rgba(64, 112, 244, 0.10);
 }
 
```

This fixes space issues in the footer for me!

```suggestion
footer > .container {
  max-width: unset;
}
```